src/style.css

```
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.star {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  border-radius: 1px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

```suggestion
.stars__star {
  margin-right: 4px;
  width: 16px;
  height: 16px;
  border-radius: 1px;
  background: url(/images/star.svg) no-repeat center;
}
```
it's better to place in that order, change to `background` and move `16px` into variable